addons/partnership/tests/test_partnership.py

```diff
@@ -2,10 +2,12 @@
 
 from odoo import Command
 from odoo.exceptions import ValidationError
+from odoo.tests import tagged
 
 from .common import PartnershipCommon
 
 
+@tagged('post_install', '-at_install')
```

Comment:
Why do you need to change that? `sale` is a dependency of this module, so this should work `at_install`, no?

Reply:
Thanks for the info. i see that the PR never got merged...
But I'm ok with both options, no problem ;-)